src/ViewInstance.js

```diff
@@ -286,6 +280,40 @@ class ViewInstance extends React.Component {
       </Button>
     );
 
+    if (query.layer === 'edit') {
+      return (
+        <Layer isOpen label={formatMsg({ id: 'ui-inventory.editInstanceDialog' })}>
```

Comment:
please follow our convention, if there is more then 1 property, place them on a separate line